### ISSUE TITLE
Migrate 'hawtio/hawtio-next' to 'Webhook To Fedora Messaging'

### ISSUE THREAD

**Issue by gridhead** [hawtio/hawtio-next#1213]
This project was listed in the [**GitHub2FedMsg**](https://github.com/fedora-infra/github2fedmsg) database and we want to you to inform you about the upcoming deprecation of the service. As the [**Fedora Infrastructure**](https://docs.fedoraproject.org/en-US/infra/) is finishing up with migrating its applications away from [**FedMsg**](https://github.com/fedora-infra/fedmsg) to [**Fedora Messaging**](https://github.com/fedora-infra/fedora-messaging), we encourage you to migrate your repository to the successor of the [**GitHub2FedMsg**](https://fedora-arc.readthedocs.io/en/latest/github2fedmsg/index.html) project, [**Webhook To Fedora Messaging**](https://webhook2fedmsg.apps.ocp.fedoraproject.org/docs).

Please follow [this link](https://communityblog.fedoraproject.org/announcing-webhook-to-fedora-messaging/) to the official announcement of the project’s release and use the instructions there to [migrate](https://pagure.io/fedora-infra/w2fm-migration/issues) to the new service. If this notification was a mistake, please close this notification ticket. We will not act on the repositories whose migrations have not been requested and any related GitHub2FedMsg operations will stop working once the service is decommissioned.

**Comment by tadayosi** [comment]
@joshiraezcode @abkieling 
I'm not sure what this actually is. I'd really appreciate it if someone can look into it and examine if there's anything that should be done from Hawtio side.

**Comment by abkieling** [comment]
@gridhead Do you know who added hawtio-next to the GitHub2FedMsg database? I am not aware of this integration.

**Comment by gridhead** [comment]
@abkieling Nope, I found the entry of this repository in the database and hence, the notification message was sent. I suspect that the addition must have happened a long time back (at least five years ago). If this is not something that the active contributors to the project know of you probably should not need it anymore.
